extensions/youtrack/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# YouTrack Changelog
+
+## [Initial Version] - 2022-07-26
+
+Icons refactoring: icons indicate if an issue is `Resolved` or `Open`.
+
+## [Initial Version] - 2022-05-23
+
+Initial version code. Support basic issues browsing.
```

Comment:
Since this is the first version of the extension being published, we can put everything under the same changelog entry. Like "## [Initial Version] - 2022-07-27" and then those two lines as bullet points